index.bs

```diff
@@ -479,6 +490,23 @@ In order to <dfn>create a WebGL depth information instance</dfn> given {{XRFrame
     1. Initialize |result|'s {{XRDepthInformation/normDepthBufferFromNormView}} to a new {{XRRigidTransform}}, based on |nativeDepthInformation|'s [=depth coordinates transformation matrix=].
     1. Initialize |result|'s {{XRWebGLDepthInformation/texture}} to an [=opaque texture=] containing the depth buffer returned in |nativeDepthInformation|.
     1. Initialize |result|'s [=XRDepthInformation/view=] to |view|.
+    1. Initialize |result|'s {{XRWebGLDepthInformation/textureType}} as follows:
+          <dl class="switch">
+            <dt> If the |result|'s {{XRWebGLDepthInformation/texture}} was created with a textureType of [=XRTextureType/texture-array=]:
+              <dd> Initialize |result|'s {{XRWebGLDepthInformation/textureType}} to "[=XRTextureType/texture-array=]".
+            <dt> Otherwise
+              <dd> Initialize |result|'s {{XRWebGLDepthInformation/textureType}} to "[=XRTextureType/texture=]".
+          </dl>
+    1. Initialize |result|'s {{XRWebGLDepthInformation/imageIndex}} as follows:
```

Comment:
This feels like it's overly specific? I certainly expect that it'll work for the majority of head-worn use cases, but I do wonder if there needs to be room left for more unique optics like the Varjo or Pimax headsets. I have no idea if they would choose to use separate depth maps for each view or not, assuming they had the ability to generate them in the first place, but it doesn't seem too far fetched.

Reply:
This is also how it's done in WebXR Layers :-)

Reply:
Eww. 😅 

That strikes me as a bug? But using common language is preferable IMO, so I'm OK with this if you could also file an issue against layers to re-evaluate that algorithm and make sure to indicate that this section should be updated to match any eventual changes in that spec's language.

Reply:
> Eww. 😅
> 
> That strikes me as a bug? But using common language is preferable IMO, so I'm OK with this if you could also file an issue against layers to re-evaluate that algorithm and make sure to indicate that this section should be updated to match any eventual changes in that spec's language.

Will do. Maybe we can add an algorithm to the layers spec that assigns the indices and refer to it from here.